docker-compose.yml

```diff
@@ -27,7 +27,9 @@ services:
     build:
       context: ./vivo
     volumes:
-      - ./example-config:/usr/local/vivo/home/config
+      - ./vivo/example.applicationSetup.n3:/usr/local/VIVO/home/config/applicationSetup.n3
+      - ./vivo/dockercompose.runtime.properties:/usr/local/VIVO/home/config/runtime.properties
```

Comment:
I've reverted the filenames to what they were before.  My reasoning is: having the outside:inside filename identical may be easier for the new user, but Vivo does use the example.applicationSetup.n3 internally.  Plus it's more difficult for old users if the filename changes from what it was previously.